backend/local/backend.go

```diff
@@ -61,6 +79,10 @@ type Local struct {
 	schema *schema.Backend
 	opLock sync.Mutex
 	once   sync.Once
+
+	// workingDir is where the State* paths should be relative to.
+	// This is currently only used for tests.
+	workingDir string
```

Comment:
Yeah, I agree. This was solely to check be able to differentiate 2 backends in a test, where I should have made a test implementation instead.